plugin-script-powershell/src/main/java/io/kestra/plugin/scripts/powershell/Command.java

```diff
@@ -0,0 +1,63 @@
+package io.kestra.plugin.scripts.powershell;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.scripts.exec.AbstractExecScript;
+import io.kestra.plugin.scripts.exec.scripts.models.DockerOptions;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.kestra.plugin.scripts.exec.scripts.services.ScriptService;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Execute one or more PowerShell commands."
+)
+public class Command extends AbstractExecScript {
+    @Schema(
+        title = "The commands to run"
+    )
+    @PluginProperty(dynamic = true)
+    @NotNull
+    @NotEmpty
+    protected List<String> commands;
+
+    @Builder.Default
+    @Schema(
+        title = "Which interpreter to use"
+    )
+    @PluginProperty
+    @NotNull
+    @NotEmpty
+    protected List<String> interpreter = List.of("/bin/sh", "-c");
+
+    @Override
+    protected DockerOptions defaultDockerOptions() {
+        return DockerOptions.builder()
+            .image("mcr.microsoft.com/powershell")
+            .entryPoint(List.of())
```

Comment:
on the commands one, people will fill raw commands, and since the powershell image forced pwsh, you can have multiple command, so I use the interpreter sh and override the entrypoint to be able to use it